unit-test/fm_child_tests.c

```diff
@@ -3512,6 +3526,55 @@ void Test_FM_ChildLoop_CountSemTakeSuccessDefault(void)
  * ChildSleepStat Tests
  * ***************/
 
+void Test_FM_ChildSleepStat_getSizeTimeModeFalse(void)
+{
+    // Arrange
+    const char *      dummy_filename        = "dummy_filename";
+    FM_DirListEntry_t dummy_DirListData     = {.EntrySize = 1, .ModifyTime = 1, .Mode = 1};
+    int32             dummy_FilesTillSleep  = 1;
+    uint8             dummy_getSizeTimeMode = 0;
+
+    // Act
+    FM_ChildSleepStat(dummy_filename, &dummy_DirListData, &dummy_FilesTillSleep, dummy_getSizeTimeMode);
```

Comment:
For unit under test functions that return void, there's a UtAssert_VOIDCALL macro.  The reason behind using it is so that the unit under test function call shows up in the log, and automated tools can parse the log and confirm all the units were called.

Reply:
Thank you. The UtAssert_VOIDCALL has been added to the FM_ChildSleepStat() tests. The two commits are squashed into 1.